tests/flow.rs

```diff
@@ -42,19 +48,48 @@ fn flowgraph_flow() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn enumerate_blocks() -> Result<()> {
+    let fg = Flowgraph::new();
+    let blocks = fg.blocks().collect::<Vec<_>>();
+    assert!(blocks.is_empty());
+    let (fg, _) = sample_fg()?;
+    let blocks = fg.blocks().collect::<Vec<_>>();
+    assert_eq!(blocks.len(), 4);
+    Ok(())
+}
+
 #[test]
 fn flowgraph_instance_name() -> Result<()> {
-    let rt = Runtime::new();
     let name = "my_special_name";
     let mut fg = Flowgraph::new();
 
     let mut source = NullSource::<f32>::new();
     let sink = NullSink::<f32>::new();
     source.set_instance_name(name);
     connect!(fg, source > sink);
-    let (_th, mut fg) = rt.start_sync(fg);
 
-    let desc = rt.block_on(async move { fg.description().await })?;
-    assert_eq!(desc.blocks.first().unwrap().instance_name, name);
+    assert!(fg
+        .blocks()
+        .find(|b| b.instance_name() == Some(name))
+        .is_some());
+
+    Ok(())
+}
+
+#[test]
+fn flowgraph_debug() -> Result<()> {
+    let (fg, _) = sample_fg()?;
+
+    // Long form
+    let dbg = format!("{:#?}", fg);
+    assert!(dbg.contains("is_blocking"), "{dbg}");
+    assert!(dbg.contains("type_name"), "{dbg}");
+
+    // Short form
+    let dbg = format!("{:?}", fg);
+    assert!(!dbg.contains("is_blocking"), "{dbg}");
+    assert!(!dbg.contains("type_name: \"Head\""), "{dbg}");
+    assert!(dbg.contains("message_edges: 0"), "{dbg}");
```

Comment:
I originally wrote this to have something to view the generated output. On the fence if it should remain.